docs/user/reconciliation.md

```diff
@@ -164,3 +164,10 @@ For subsequent updates, the builder blocks until:
 ### Job
 
 After pushing the merged Job into the cluster, the builder blocks until the Job succeeds.
+
+Since neither the use case requiring a Job's spec.selector to be explicitly set nor the handling thereof is well defined the code will panic if a user attempts to set spec.selector.
+
+Subsequent updates:
+
+* Job update handling is not well-defined and is therefore not recommended.
```

Comment:
Is this getting at "we'll probably want to delete/recreate jobs on changes, but don't have code for that yet"?  If so, maybe rephrase to:

> Current limitations:
>
> * The cluster-version operator is currently unable to delete and recreate a job to track changes in release manifests.  Please avoid making changes to Job manifests until it grows that support.

or some such.

Reply:
6b58e7a wording looks good to me.